src/gridstack-dd.ts

```diff
@@ -25,7 +25,7 @@ export type DDKey = 'minWidth' | 'minHeight' | 'maxWidth' | 'maxHeight';
 export type DDValue = number | string;
 
 /** drag&drop events callbacks */
-export type DDCallback = (event: Event, arg2: GridItemHTMLElement, helper?: GridItemHTMLElement) => void;
+export type DDCallback = GridStackEventHandlerCallback;

```

Comment:
no point in defining a type from another type!

Reply:
Does DDCallback need to be its own type? It should be the same type as GridStackEventHandlerCallback so I was hoping we could share the types

Reply:
not if it's the same, but it is different.